policy-engine/src/main.rs

```diff
@@ -141,7 +170,8 @@ async fn main() -> Result<(), Error> {
     .run();
 
     BUILD_INFO.inc();
-
+    // application has initialized.
+    *state.ready.write() = true;
```

Comment:
We probably want to wait until we've synchronized our graph-data cache before going ready, because once we enter the load balancer, we will begin serving client requests.  Skimming the above^^, I'm not clear on whether or not that's happened by this point.

Reply:
added additional logic to wait for the graph to refresh and cache before going ready